tests/sql/test_aggs.py

```diff
@@ -41,3 +44,39 @@ def test_aggs_sql():
     )
 
     assert actual == expected
+
+
+@pytest.mark.parametrize(
+    "agg,cond,expected",
+    [
+        ("sum(values)", "sum(values) > 10", {"values": [20.5, 29.5]}),
+        ("sum(values)", "values > 10", {"values": [20.5, 29.5]}),
+        ("sum(values) as sum_v", "sum(values) > 10", {"sum_v": [20.5, 29.5]}),
+        ("sum(values) as sum_v", "sum_v > 10", {"sum_v": [20.5, 29.5]}),
+        ("count(*) as cnt", "cnt > 2", {"cnt": [3, 5]}),
+        ("count(*) as cnt", "count(*) > 2", {"cnt": [3, 5]}),
+        ("count(*)", "count(*) > 2", {"count": [3, 5]}),
```

Comment:
Add some test cases here where the having agg is not in the select, as well as some where having uses the group by key.